Tests/MapboxSearchUITests/Integration/FavoritesIntegrationTestCase.swift

```diff
@@ -1,6 +1,7 @@
 import XCTest
 
-class FavoritesIntegrationTestCase: MockSBSServerUITestCase {
+// Rename to UITestCase
+class FavoritesIntegrationTestCase: MockSearchBoxUITestCase {
```

Comment:
Favorites UI tests are failing on Minsk name in results and this will be fixed in SAPI-1176